setup.cfg

```diff
@@ -47,14 +18,15 @@ plottings =
     %(plotting)s
 test =
```

Comment:
There's no way to do these dynamic `extras_requires` in `pyproject.toml`, so I left them. Also left metadata and options that are setuptools-specific, as those raise warnings that the feature is in beta for the time being.